docs/content/doc/administration/https-support.en-us.md

```diff
@@ -40,7 +40,7 @@ KEY_FILE  = key.pem
 ```
 
 Note that if your certificate is signed by a third party certificate authority (i.e. not self-signed), then cert.pem should contain the certificate chain. The server certificate must be the first entry in cert.pem, followed by the intermediaries in order (if any). The root certificate does not have to be included because the connecting client must already have it in order to estalbish the trust relationship.
-To learn more about the config values, please checkout the [Config Cheat Sheet](../config-cheat-sheet#server-server).
+To learn more about the config values, please checkout the [Config Cheat Sheet]({{< relref "doc/administration/config-cheat-sheet.en-us.md#server-server" >}}).
```

Comment:
I am not sure if you can link to URL fragments inside `relref`.
I don't think you can?